### PR TITLE
#113 SCS MultiAPI Plugin get yaml files from plugin dependencies

### PR DESCRIPTION
**Pull request by 5uso-sng**
Solves #113 and #215. The updated readme explains how to use the new feature.

### CODE REVIEW

File: multiapi-engine/src/main/java/com/sngular/api/generator/plugin/openapi/OpenApiGenerator.java [hunk 1]
```diff
@@ -247,10 +246,10 @@ private String convertPackageToTargetPath(final String fileSpecPackage, final Bo
     if (StringUtils.isNotBlank(fileSpecPackage)) {
       path = FilenameUtils.concat(processedGeneratedSourcesFolder, PACKAGE_SEPARATOR.matcher(fileSpecPackage.trim()).replaceAll("/"));
     } else if (groupId != null) {
-      path = FilenameUtils.concat(processedGeneratedSourcesFolder, PACKAGE_SEPARATOR.matcher(groupId).replaceAll(File.separator));
+      path = FilenameUtils.concat(processedGeneratedSourcesFolder, PACKAGE_SEPARATOR.matcher(groupId).replaceAll("/"));
     } else {
       final String pathDefault = Boolean.TRUE.equals(isModel) ? DEFAULT_OPENAPI_MODEL_PACKAGE : DEFAULT_OPENAPI_API_PACKAGE;
-      path = FilenameUtils.concat(processedGeneratedSourcesFolder, PACKAGE_SEPARATOR.matcher(pathDefault).replaceAll(File.separator));
+      path = FilenameUtils.concat(processedGeneratedSourcesFolder, PACKAGE_SEPARATOR.matcher(pathDefault).replaceAll("/"));
```
Comment (josemenvegsngular) [inline multiapi-engine/src/main/java/com/sngular/api/generator/plugin/openapi/OpenApiGenerator.java:252]: Why this change? we use the same constant three times in very few lines.
We can even think in creating a private function like:
```
private <what ever is path> aGoodName(<processedGeneratedSourcesFolder if it's needed to send as a parmeter>, string name) {
  return FilenameUtils.concat(processedGeneratedSourcesFolder, PACKAGE_SEPARATOR.matcher(name).replaceAll("/"));
}
```

Comment (5uso-sng) [reply multiapi-engine/src/main/java/com/sngular/api/generator/plugin/openapi/OpenApiGenerator.java:252]: This changes the literal `\` for `/`, as it was causing crashes due to being an escape character in regex. I'll simplify the flow while we're at it

**Review by josemenvegsngular — DISMISSED** [review]

**Review by davidgayoso — APPROVED** [review]
Everything seems to be good. Nice job!

**Review by janDuinRod — APPROVED** [review]
Everything looks good

**Review by davidgayoso — APPROVED** [review]
